server/controllers/UsersController.js

```diff
@@ -1,8 +1,11 @@
 import bcrypt from 'bcrypt-nodejs';
+import jwt from 'jsonwebtoken';
 import models from '../models';
 import createToken from '../helpers/createToken';
 import sendEmail from '../helpers/sendEmail';
 import verifyEmailMessage from '../helpers/verifyEmailMessage';
+import resetPasswordEmail from '../helpers/resetPasswordEmail';
+import forgotPasswordEmail from '../helpers/forgotPasswordEmail';
```

Comment:
Seeing that both _resetPasswordEmail_ and _forgotPasswordEmail_ are basically functions serving as templates for the possible emails we need to send, you could have all possible templates in one file, each one exported. That way, you can import (destructure) both functions from the same file.

import { forgotPasswordEmail, resetPasswordEmail } from '../helpers/emailTemplates';

Then your emailTemplates.js can have something similar to what is below.
```
export const forgotPasswordEmail = resetUrl => ({
  subject: 'Authors Haven Password Reset',
  body:
    `<p>
      Click <a href="${resetUrl}">here</a> to reset your password.
    </p>`
});
```

```
export const resetPasswordEmail = () => ({
  subject: 'Authors Haven Password Reset',
  body:
    `<p>
      Your password has successfully been reset.
    </p>`
});
```

Reply:
Okay, I will fix this.